Tasks/GitHubReleaseV0/task.json

```diff
@@ -265,6 +265,10 @@
         "InvalidEndpointAuthScheme": "Invalid GitHub service connection scheme: %s. Only OAuth and GitHub personal access token connections are allowed.",
         "FetchInitialCommitError": "An unexpected error occurred while fetching the initial commit.",
         "InvalidActionSet": "Invalid action: %s. Only 'create', 'edit', or 'delete' actions are allowed.",
+        "InvalidTagSource": "Invalid tag source: %s. Only 'auto', or 'manual' options are allowed.",
+        "InvalidReleaseNotesSource": "Invalid release notes source: %s. Only 'file', or 'input' options are allowed.",
+        "InvalidAssetUploadMode": "Invalid asset upload mode: %s. Only 'delete', or 'replace' options are allowed.",
+        "TagRequired": "For edit/delete action or create action with tag source set to manual, a tag is required. Please specify a tag in the step",
```

Comment:
split the error message. for Edit/Delete pass action from context. For create with tag source ..one hard coded message